packages/features/ee/teams/lib/payments.ts

```diff
@@ -59,9 +62,10 @@ export const generateTeamCheckoutSession = async ({
     customer_update: {
       address: "auto",
     },
-    automatic_tax: {
-      enabled: true,
-    },
+    // You can comment it out when testing locally as usually developer doesn't setup Tax in Stripe Test mode
+    // automatic_tax: {
+    //   enabled: true,
+    // },
```

Comment:
Would it make sense to check this in a conditional -> `IS_PRODUCTION ? {...} : {}`

Reply:
Yeah it would. I wasn't sure which variable to base the condition on.